include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -55,6 +55,16 @@ def InstanceOp : ReferableDeclOp<"instance", [HasParent<"firrtl::FModuleOp, firr
                    CArg<"ArrayRef<Attribute>", "{}">:$portAnnotations,
                    CArg<"bool","false">:$lowerToBind,
                    CArg<"StringAttr", "StringAttr()">:$inner_sym)>,
```

Comment:
Are these builders (StringAttr ones) expected to be removed in the future, or will they grow variants for visibility, etc.?  Just curious.